### ISSUE TITLE
Namco Museum hangs w/ black screen when NoAudioDelay is true in mupen64plus.conf

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jennorby/mupen64plus#48]
```
- Gentoo linux
- Affects 32-bit and 64-bit builds
- SVN rev 186 and older


```

Original issue reported on code.google.com by `richard...@gmail.com` on 13 Apr 2008 at 5:14

**Comment by GoogleCodeExporter** [comment]
```
This has most likely something to do with timing issues, and is not really an 
issue
as audio sync is sometimes required for compatibility. But it might be 
interesting to
take a look at.
```

Original comment by `sgorman07@gmail.com` on 14 Apr 2008 at 4:47

**Comment by GoogleCodeExporter** [comment]
```
The NoAudioDelay core option was removed from Mupen64Plus in rev 496 to resolve 
this
issue.  The "readme.pdf" included with Mupen64 v0.5 states that this option 
(along
with NoCompiledJump) were not necessary for v0.5, so presumably this option can 
be
safely removed.
```

Original comment by `richard...@gmail.com` on 3 Jun 2008 at 3:47
- Changed state: **Fixed**
